docs/PRD.md

```
@@ -294,23 +294,22 @@
 
 #### 7.2. Tri-shot 연속 촬영 (Remote Shooting)
 
--   **설명:** `MainView`의 Segmented Control을 통해 접근하는 원격 촬영 기능입니다. 사용자가 정의한 여러 촬영 설정(프리셋)을 한 번의 셔터로 연속 촬영하게 합니다.
+-   **설명:** `MainView`의 Segmented Control을 통해 접근하는 원격 촬영 기능입니다. 사용자가 정의한 여러 촬영 설정(프리셋)을 반복적으로 촬영하게 됩니다.
 -   **주요 기술:**
     -   **네트워킹 (`URLSession`, `Digest Authentication`):** Canon 카메라와의 실시간 통신(CCAPI)을 담당합니다.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)